src/metadata/metadata.rs

```diff
@@ -58,6 +63,14 @@ impl Metadata {
                 let val = val.to_owned();
                 self.raw_status = Some(val);
             }
+            "Boilerplate" => {
+                let val = match parse::parse_boilerplate(val) {
+                    Ok(val) => val,
+                    Err("wrong boolish format") => die!(""; line_num),
```

Comment:
Given the two branches are equal, is there any reason you need this? Why can't we remove this line?